### DISCUSSION TITLE
Issue on mobile device with menu

### DISCUSSION THREAD

**Discussion by JShaharov** [netbox-community/netbox#14602] [General]
Loving netbox, but encountered a weird issue.  On my mobile devices, using android and chrome web browser, the menu on the left doesnt appear, making it very hard to navigate.  Does anyone have any ideas?
![Screenshot_20231225-025551_Chrome](https://github.com/netbox-community/netbox/assets/29449454/92abf2a6-b394-42cc-880e-5104091e7dd8)

**Comment by candlerb** [comment]
Please describe exactly what "mobile device(s)" you are referring to. Is it some sort of a tablet? What resolution? Is that screenshot you posted from the mobile device in question?

Also, what version of Netbox are you using? And what versions of Android and Chrome?

On a phone with a narrow screen, Netbox shows a hamburger menu in place of the LH menu. This dynamically adjusts: you can check this on a desktop machine, just drag the window narrower until it flips to mobile mode.

Maybe you've found an odd corner situation with the particular resolution of your mobile device; or maybe the CSS fallback isn't working properly on that device.

**Reply by JShaharov** [reply]
It was on an older samsung Tablet.  I tried on my phone so it must be just a bug somewhere.  For information its running lineage  and android 9.  Im not going to worry too much about it as I just wont use the tablet

**Reply by candlerb** [reply]
Android 9 isn't *that* ancient (released Dec 2018, last security update Jan 2022), but it wouldn't surprise me if there's some bit of Netbox CSS which doesn't quite work with it.

**Comment by JShaharov** [comment]
Well, it does work on my phone, which is android 9 as well so I dont think
its the OS.  Both have the latest version of chrome browser.  Maybe due to
resolution.  Not sure

Jonathan Shahariw
-------------------------------
